modules/ROOT/pages/tools/cypher-shell.adoc

```diff
@@ -114,6 +115,10 @@ It can also be set using the environmental variable NEO4J_CYPHER_SHELL_HISTORY.
 |--notifications
 |Enable query notifications in interactive mode.
 |false
+
+|--idle-timeout IDLE-TIMEOUT
+|Idle timeout, when set the application will exit after the specified amount of idle time in interactive mode. Specify with `<hours>h<minutes>m<seconds>s`, for example `1h` (1 hour), `1h30m` (1 hour 30 minutes), `30m` (30 minutes).
```

Comment:
```suggestion
|Closes the application after the specified amount of idle time in interactive mode. You can specify the duration using the format `<hours>h<minutes>m<seconds>s`, for example `1h` (1 hour), `1h30m` (1 hour 30 minutes), or `30m` (30 minutes).
```

Reply:
If you use this description, please also update the code.

Reply:
Sweet, thanks!